$MirOS: ports/audio/libvorbis/patches/patch-configure_in,v 1.2 2005/11/09 19:44:59 bsiegert Exp $
$OpenBSD: patch-configure,v 1.6 2003/12/14 00:52:22 naddy Exp $
--- configure.in.orig	Mon Nov 28 05:43:03 2005
+++ configure.in	Wed Dec 28 16:00:04 2005
@@ -93,7 +93,7 @@ if test -z "$GCC"; then
 		PROFILE="-v -xpg -g -xO4 -fast -native -fsimple -xcg92 -Dsuncc" ;;
 	*)
 		DEBUG="-g"
-		CFLAGS="-O"
+		CFLAGS=""
 		PROFILE="-g -p" ;;
         esac
 else
@@ -164,9 +164,9 @@ else
 		CFLAGS="-DDARWIN -fno-common -force_cpusubtype_ALL -Wall -g -O4 -ffast-math -fsigned-char"
 		PROFILE="-DDARWIN -fno-common -force_cpusubtype_ALL -Wall -g -pg -O4 -ffast-math -fsigned-char";;
 	*)
-		DEBUG="-g -Wall -W -D__NO_MATH_INLINES -fsigned-char"
-		CFLAGS="-O20 -D__NO_MATH_INLINES -fsigned-char"
-		PROFILE="-O20 -g -pg -D__NO_MATH_INLINES -fsigned-char" ;;
+		DEBUG="-g -Wall -W -fsigned-char"
+		CFLAGS="-fsigned-char"
+		PROFILE="-g -pg -fsigned-char" ;;
         esac
 fi
 CFLAGS="$CFLAGS $cflags_save"
